modules/cnf-fast-event-notifications-api-refererence.adoc

```diff
@@ -7,11 +7,18 @@
 
 You can use the PTP fast event notifications REST API to subscribe an application to the PTP events that are generated for the parent node. PTP fast events notifications are available on each node where a PTP capable network interface is configured.
 
-You can subscribe DU applications to PTP event notifications by using the resource address `/cluster/node/<node_name>/ptp`, where `<node_name>` is the cluster node running the DU application.
+You can subscribe distributed unit (DU) applications to PTP event notifications by using the resource address `/cluster/node/<node_name>/ptp`, where `<node_name>` is the cluster node running the DU application.
 
-Status request are sent by making a REST API `PUT` call to `/subscriptions/status/<subscription_id>`. The API call returns events through the AMQP event bus to the subscribed address.
+The PTP Operator deploys the `linuxptp-daemon` container and the `cloud-event-proxy` sidecar container in a pod managed by the PTP Operator. The `cloud-event-proxy` container collects the PTP events and publishes the events using the `cloud-event-proxy` publisher API at `http://localhost:8089/api/cloudNotifications/v1/`.
 
-The PTP fast events notifications REST API is served at `http://localhost:8080/` by default. The following API endpoints are available:
+Deploy your DU application container that needs to subscribe to PTP events and a `cloud-event-proxy` sidecar container in a separate DU application pod. The DU application container uses the subscription API provided by the `cloud-event-proxy` container to subscribe to events at `http://localhost:8089/api/cloudNotifications/v1/`.
+
+[NOTE]
+====
+The PTP fast events notifications REST API is provided by link:https://github.com/redhat-cne/cloud-event-proxy[`cloud-event-proxy`]. To access the API, applications must be in the same pod as the `cloud-event-proxy` container.
```

Comment:
We need to have GitHub links reviewed/approved by Eric Rich. If this is critical for GA, I'd just remove the link for now and ask later/add back in if approved.